projects/schematics/src/shared/constants.ts

```diff
@@ -866,6 +866,8 @@ export const LOGIN_MODULE = 'LoginModule';
 export const LOGIN_FORM_MODULE = 'LoginFormModule';
 export const LOGIN_REGISTER_COMPONENT = 'LoginRegisterComponent';
 export const LOGIN_REGISTER_MODULE = 'LoginRegisterModule';
+export const MYACCOUNT_V2_USER_MODULE = 'MyAccountV2UserModule';
+
```

Comment:
[NITPICK]
I have problem to find this const being used anywhere in the codebase

Reply:
removed